packages/css/test/index.ts

```diff
@@ -160,6 +160,25 @@ test('handles all core styled system props', () => {
   })
 })
 
+test('handles css logical properties', () => {
```

Comment:
This feels a bit nitpicky, but I think it'd be worthwhile to include all the logical properties in a test since any one of them could regress.